src/main/resources/application.yml

```diff
@@ -1 +1,32 @@
+logging:
+  level:
+    root: INFO
+    org.springframework.web: INFO
+    org.springframework.security: INFO
+    org.springframework.security.oauth2: INFO
+    org.springframework.boot.autoconfigure: DEBUG
```

Comment:
I would prefer to have the logging configuration separated from the application configuration - is this possible without much hassle?

Reply:
Currently we don't have the logging infrastructure yet, this is just here to help us with debugging. Later, patches will be added to address logging.